code/modules/planet/weather.dm

```diff
@@ -0,0 +1,268 @@
+#define WEATHER_CLEAR				"clear"
+#define WEATHER_OVERCAST			"overcast"
+#define WEATHER_LIGHT_SNOW			"light snow"
+#define WEATHER_SNOW				"snow"
+#define WEATHER_BLIZZARD			"blizzard"
+#define WEATHER_RAIN				"rain"
+#define WEATHER_STORM				"storm"
+#define WEATHER_HAIL				"hail"
+#define WEATHER_WINDY				"windy"
+#define WEATHER_HOT					"hot"
+
+/datum/weather_holder
+	var/datum/planet/our_planet = null
+	var/datum/weather/current_weather = null
+	var/temperature = T20C
+	var/wind_dir = 0
+	var/wind_speed = 0
```

Comment:
Don't think these winds are ever checked or modified either, are we going to implement that at some point?